WordPress/Classes/Models/WPAccount.m

```diff
@@ -116,14 +116,11 @@ - (NSArray *)visibleBlogs
     return sortedBlogs;
 }
 
-#pragma mark - API Helpers
+#pragma mark - WordPress.com support methods
 
-- (WordPressComApi *)restApi
+- (BOOL)isWPComAccount
 {
-    if (!_restApi) {
-        _restApi = [[WordPressComApi alloc] initWithOAuthToken:self.authToken];
-    }
-    return _restApi;
+    return self.restApi != nil;
```

Comment:
Good call.  This should be configured in the initializer probably as well.  I'll try options.